fairlearn/datasets/_synthetic.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+import numpy as np
+from sklearn.datasets import make_classification
+from sklearn.utils import check_random_state
+
+
+def make_synthetic_dataset(classes=None, n_features=20, n_informative=4, random_state=None):
+    """Create a synthetic dataset with a single sensitive feature: 'gender'.
+
+    Parameters
+    ----------
+    classes : dict, default=None
+        A dict whose keys are the labels for the gender feature, and whose set
+        the `n_samples` and `class_sep` parameters for `make_classification`.
+        If `None`, the following default dict is used::
+
+            {
+                'Man': {'n_samples': 500, 'class_sep': 1},
+                'Other': {'n_samples': 500, 'class_sep': 0.5},
+                'Unspecified': {'n_samples': 500, 'class_sep': 0.5},
+                'Woman': {'n_samples': 500, 'class_sep': 2},
+            }
+
+    n_features : int, default=20
+        The total number of features.
+
+    n_informative : int, default=4
+        The number of informative features.
+
+    random_state : int or RandomState instance, default=None
+        The random number generator seed or RandomState to use.
```

Comment:
```suggestion
        The random number generator seed or :class:`numpy.random.RandomState` to use.
```
Note that this is only needed in this line for the description, since the line above (the type annotation) doesn't require the ReST directive.

Reply:
Fixed, thanks!